src/hooks/supabase-swr/readme.md

```diff
@@ -0,0 +1,3 @@
+Basically an inline fork of https://github.com/alfredosalzillo/supabase-swr
```

Comment:
This folder started as a fork. Have made a lot of tweaks and clean up. I removed the `useSession` as we are not using it and it needed work. Eventually it will go back in. Making plans to pull this out to a stand alone repo soon.